cirq/testing/consistent_act_on.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2019 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any
+
+import numpy as np
+
+from cirq.circuits.circuit import Circuit
+from cirq.devices import LineQubit
+from cirq.ops import common_gates
+from cirq.ops.dense_pauli_string import DensePauliString
+from cirq import protocols
+from cirq.sim import act_on_state_vector_args, final_state_vector
+from cirq.sim.clifford import act_on_clifford_tableau_args, clifford_tableau
+
+
+def state_vector_has_stabilizer(state_vector: np.ndarray,
```

Comment:
This method currently mutates the `state_vector` argument and makes a copy that it doesn't mutate. Switch it around so the input argument is not mutated but the copy is mutated.

Reply:
Done